x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/three_way_diff/comparison_side/field_comparison_side.stories.tsx

```diff
@@ -44,14 +44,8 @@ interface TemplateProps<FieldName extends keyof DiffableAllFields> {
   resolvedValue?: DiffableAllFields[FieldName];
 }
 
-const Template: Story<TemplateProps<keyof DiffableAllFields>> = (args) => {
-  return (
-    <ComparisonSide
-      fieldName={args.fieldName}
-      fieldThreeWayDiff={args.fieldThreeWayDiff}
-      resolvedValue={args.resolvedValue}
-    />
-  );
+const Template: Story<TemplateProps<keyof DiffableAllFields>> = () => {
+  return <FieldComparisonSide />;
```

Comment:
<img width="1392" alt="Scherm­afbeelding 2024-12-05 om 16 23 52" src="https://github.com/user-attachments/assets/dfe9b851-0667-47e2-9d6a-e29ed6135ab4">

We also need to wrap the component with the provider, since it uses context now.

Reply:
I thought maybe we should remove storybooks since we have a real environment right now. WDYT?

Reply:
I really don't have a strong argument to have them anymore. They are more of maintenance burden now. Let's discuss with the team. I'll add a topic.